### ISSUE TITLE
Missing Data Processing and Evaluation Script

### ISSUE THREAD

**Issue by Anderson-Lee-Git** [SunGL001/OGPSA#1]
Hello @SunGL001,

Cool work on the OGPSA. Would you mind sharing the evaluation and the data preprocessing script? It's unclear which seed was used to draw the random 10K samples from the SafeRLHF dataset, and the use of gpt-4o-mini to generate the response makes it difficult to reproduce the results without access to the processed SFT dataset. Thank you so much!

**Comment by Anderson-Lee-Git** [comment]
Hello @SunGL001, 

Thank you so much for updating the repository and including the training dataset. Would you mind pointing out the source of HHH dataset used in the evaluation? It was stated in the paper that HHH is the big-bench, but big-bench has several variants, and the citation the paper points to doesn't seem to have the dataset. Thank you so much!
